shuttle/rpc/engines/websocket/websocket.go

```diff
@@ -107,9 +108,18 @@ func (m *manager) Connect(ws *websocket.Conn, handle string, done chan struct{})
 		QueueEngEnabled:       hello.QueueEngEnabled,
 	}
 
+	err = nil
 	m.shuttlesLk.Lock()
-	m.shuttles[handle] = sc
```

Comment:
if it already exists, would this not replace it? also, we do not want to fail, we want to replace it with the latest connection. Except we are saying `m.shuttles[handle] = sc` is not garbage collected

Reply:
Makes sense. Instead of this code change should we call a clean up command on the old connection we are replacing?

Reply:
something like that, I was of the assumption just setting the same handle does garbage collection

Reply:
lets try out bytes first